File: src/main/java/service/task/management/TaskManagementApplication.java
package service.task.management;

import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@Slf4j
@SpringBootApplication
public class TaskManagementApplication {

	public static void main(String[] args) {
		log.info("Application started");

		SpringApplication.run(TaskManagementApplication.class, args);
	}

}

File: src/main/kotlin/service/task/management/TaskManagementApplication.kt
package service.task.management

import dev.misfitlabs.kotlinguice4.KotlinModule
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import service.task.management.service.ServiceModule

@SpringBootApplication
class TaskManagementApplication: KotlinModule() {
    override fun configure() {
        install(ServiceModule())
    }
}

fun main(args: Array<String>) {
    runApplication<TaskManagementApplication>(*args)
}
